Dockerfile-dev

```
@@ -0,0 +1,6 @@
+FROM openjdk:17-jdk-slim
+ARG JAR_FILE=build/libs/app.jar
+COPY ${JAR_FILE} app.jar
+
+# 런타임 환경변수 사용을 위해 sh -c 사용
+ENTRYPOINT ["sh", "-c", "java -Duser.timezone=Asia/Seoul -Dspring.profiles.active=${PROFILE} -jar /app.jar"]
```

배포 환경에서 .env 파일 누락 등 실수 방지를 위해 PROFILE 기본값을 지정하는 것이 좋아보입니다.

```suggestion
ENTRYPOINT ["sh", "-c", "java -Duser.timezone=Asia/Seoul -Dspring.profiles.active=${PROFILE:-dev} -jar /app.jar"]
```

---

```
ENTRYPOINT ["sh", "-c", "java -Duser.timezone=Asia/Seoul -Dspring.profiles.active=${PROFILE} -jar /app.jar"]

```

여기서 보시면 ${PROFILE} 이라는 환경 변수를 가져오는데, 이 환경변수를 어디서 주입해주냐면 `docker-compose.dev.yml`에서 주입해줍니다!